substrate/bin/node/cli/src/chain_spec.rs

```diff
@@ -22,10 +22,11 @@ use polkadot_sdk::*;
 
 use crate::chain_spec::sc_service::Properties;
 use kitchensink_runtime::{
-	genesis_config_presets::{session_keys, Staker, STASH},
+	genesis_config_presets::{Staker, ENDOWMENT, STASH},
 	wasm_binary_unwrap, Block, MaxNominations, StakerStatus,
 };
```

Comment:
Am I correct that we want to remove all imports from the runtime here except for `kitchensink_runtime::WASM_BINARY`? Aka, this would imply that wherever possible we move the definitions to the `node-primitives` and in some cases there might be duplicated code.